elixir/apps/api/lib/api/client/views/interface.ex

```diff
@@ -10,11 +11,23 @@ defmodule API.Client.Views.Interface do
         Map.from_struct(%{dns_config | address: address})
       end)
 
+    upstream_do53 =
+      for %{protocol: :ip_port, address: address} <-
+            Map.get(client.account.config, :clients_upstream_dns, []),
+          {:ok, ip_port} <- [Domain.Types.IPPort.cast(address)] do
```

Comment:
Does this ever match? The left side is a tuple and the right a list.

Reply:
Going to push a commit to tidy this up.

Reply:
Pushed a commit.

For the new field, we don't want to use IPPort at all since we've already earmarked that for removal as it's essentially dead code.

I believe you mentioned we don't use the port for the old field either? Could we leave that out or is it required for parsing consistency?